pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+# Description
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+`Your message here`
+
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Other, Please explain:
+`Your message here`
```

Comment:
Can we add a checkbox for non-breaking styling changes?

Reply:
done!